provision/kubernetes/pkg/client/clientset/versioned/fake/clientset_generated.go

```diff
@@ -62,7 +62,7 @@ func (c *Clientset) Tracker() testing.ObjectTracker {
 	return c.tracker
 }
 
-var _ clientset.Interface = &Clientset{}
+var _ clientset.Interface = (*Clientset)(nil)
```

Comment:
This code is automatically generated by kubernetes scripts. It should not be changed.